rollup.config.js

```diff
@@ -23,12 +23,12 @@ export default {
       // we'll extract any component CSS out into
       // a separate file — better for performance
       css: css => {
-        css.write('public/bundle.css');
+        css.write('bundle.css');
```

Comment:
Why do we have to remove `public` from here now? 🤔

Reply:
The compiler is automatically adding "public/" to the path so without removing it, the file will be compiled to "public/public/bundle.css".